cmd/cozystack-controller/main.go

```diff
@@ -39,6 +39,8 @@ import (
 	cozystackiov1alpha1 "github.com/cozystack/cozystack/api/v1alpha1"
 	"github.com/cozystack/cozystack/internal/controller"
 	"github.com/cozystack/cozystack/internal/telemetry"
+
+	helmv2 "github.com/fluxcd/helm-controller/api/v2beta1"
```

Comment:
Let's use [v2](https://github.com/fluxcd/helm-controller/tree/main/api/v2)

```suggestion
	helmv2 "github.com/fluxcd/helm-controller/api/v2"
```